appveyor.yml

```diff
@@ -0,0 +1,37 @@
+# AppVeyor CI settings (Windows Machine CI Tests)
+
+environment:
+  matrix:
+    - python: "2.7"
+      env: TOXENV=lint-py2,docs
+    - python: "3.6"
+      env: TOXENV=lint-py3,docs
+    - python: "2.7"
+      env: TOXENV=py27
+    - python: "2.7"
+      env: TOXENV=py27-rp3
+    - python: "2.7"
+      env: TOXENV=py27-datetime
+    - python: "3.4"
+      env: TOXENV=py34
+    - python: "3.5"
+      env: TOXENV=py35
+    - python: "3.6"
+      env: TOXENV=py36
+    - python: "3.6"
+      env: TOXENV=py36-datetime
+    - python: "3.7-dev"
+      env: TOXENV=py37
+
+cache:
+  - '%LOCALAPPDATA%\pip\Cache'
+
+version: '{branch}.{build}'
+
+install:
+  - "%PYTHON%\\python.exe -m pip install tox"
+
+build: off
+
+test_script:
+  - "%PYTHON%\\Scripts\\tox.exe"
```

Comment:
I remember that I have seen test running on Appveyor but the PR does not list it. Has there still something to be done?

Reply:
Appveyor needs an appveyor.yml in the master branch to understand when it should run, so that will do it in the future. I have already enabled it.